lucky-number-model/Models/LuckyNumber.cs

```diff
@@ -8,14 +8,61 @@ namespace lucky_number_model.Models
 {
     public class LuckyNumber
     {
+        private decimal _balance;
+        private Boolean _iswinner;
+        private int[] _spinner = new int[3];
+
+        private Random _rnd = new Random();
+
         [Required]
         [Display(Name = "Lucky Number")]
         [Range(1, 9, ErrorMessage = "Number must be between 1 and 9")]
         public int Number { get; set; } // the user's choice of a lucky number
 
+        // the balance for this game
         [Required]
-        public decimal Balance { get; set; } // the balance for this game
+        public decimal Balance
+        {
+            get
+            {
+                if (isWinner) { _balance = _balance + 2; }
```

Comment:
Inside the Model, use the instance variables like "_isWinner", instead of the getter "isWinner"